doc/reference/tooling/tt_cli/pack.rst

```diff
@@ -76,6 +76,48 @@ An example of the ``systemd-unit-params.yml`` file:
       INSTANCE: "inst:%i"
       TARANTOOL_WORKDIR: "/tmp"
 
+.. _tt-pack-integrity-check:
+
+Generating files for integrity checks
+-------------------------------------
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    The integrity check functionality is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+``tt pack`` can generate hashes and signatures to use for integrity checks
+when running the application. These files are:
+
+-   ``hashes.json`` and ``hashes.json.sig`` in each application directory.
+    ``hashes.json`` contains SHA256 checksums of executable files that the application uses
+    and its configuration file. ``hashes.json.sig`` contains a digital signature
+    for ``hashes.json``.
+
+-   ``env_hashes.json`` and ``env_hashes.json.sig`` in the environment root are
+    similar files for the ``tt`` environment. They contain checksums for
+    Tarantool and ``tt`` executables, and for the ``tt.yaml`` configuration file.
+
+To generate hashes and signatures for integrity check, use the ``--with-integrity-check``
+option. Its argument must be an RSA private key.
+
+.. note::
+
+    You can generate a key pair using `OpenSSL <https://www.openssl.org/>`__  as follows:
```

Comment:
It's better to specify the version since major versions of openssl could have an incompatible differences.
```suggestion
    You can generate a key pair using `OpenSSL 3 <https://www.openssl.org/>`__  as follows:
```